pkg/daemon/ceph/client/pool_test.go

```diff
@@ -203,3 +204,46 @@ func TestGetPoolStatistics(t *testing.T) {
 	assert.NotNil(t, err)
 	assert.Nil(t, stats)
 }
+
+func TestSetPoolReplicatedSizeProperty(t *testing.T) {
```

Comment:
What about the case where pool size is 1 and `allowUnsafeSingleReplica=false`?

Reply:
This is tested here: https://github.com/rook/rook/pull/4895/files#diff-13691686b2667ae3af0c97b55a36fd64R65